tasks/custom_facts.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Create Ansible facts directory
+  file:
+    path: "/etc/ansible/facts.d"
+    state: "directory"
+  check_mode: no
+
+- name: NUMA node fact
+  copy:
+    content: |
+      #!/bin/bash
+      export LC_ALL=en_GB.UTF8
+      for socket in $(/usr/bin/lscpu --all --parse=node,socket | grep -E "[0-9]$" | cut -d, -f2 | sort | uniq); do
+        echo "[$socket]"
+        echo "nodes="$(/usr/bin/lscpu --all --parse=node,socket | grep -E "${socket}$" | cut -d, -f1 | sort -n | uniq | paste -sd ',' -)
+        echo
+      done
+    dest: "/etc/ansible/facts.d/numanodes.fact"
```

Comment:
This will potentially overwrite or be overwritten by the `numanodes.fact` in the Humio role. So either we should just change the name, or extract it into a "custom facts" role/module?

Reply:
It might make sense to roll this into a separate role/module, but in the interest of keeping things simple (and keeping total role count down), I think I'll just rename the file kafka drops. Both files should be identical at this point, but better safe than sorry.